include/lsst/afw/image/detail/StorableMap.h

```diff
@@ -68,6 +70,23 @@ class StorableMap final {
     using pointer = value_type*;
     using const_pointer = value_type const*;
 
+private:
+    // Can't map by full Key because we don't have a Key comparator that
```

Comment:
Should this private block be moved to the bottom?

Reply:
It should, but it can't. You need at least a signature for `_toTypedKey` when `const_iterator` is first declared.